pkgs/development/python-modules/torch/default.nix

```diff
@@ -436,11 +436,7 @@ in buildPythonPackage rec {
     blasProvider = blas.provider;
     # To help debug when a package is broken due to CUDA support
     inherit brokenConditions;
-  } // lib.optionalAttrs cudaSupport {
-    # NOTE: supportedCudaCapabilities isn't computed unless cudaSupport is true, so we can't use
-    #   it in the passthru set above because a downstream package might try to access it even
-    #   when cudaSupport is false. Better to have it missing than null or an empty list by default.
-    cudaCapabilities = supportedCudaCapabilities;
+    cudaCapabilities = if cudaSupport then supportedCudaCapabilities else [ ];
```

Comment:
@ConnorBaker objections?

Reply:
Eh, no objections here -- in retrospect, perhaps it *is* better to have an empty list than for it to just be missing.

We don't have a common API or way of exposing which capabilities a derivation is built with. Maybe something to look into for the future?